File: app/src/main/java/br/dpacanhella/miguelopolis/data/model/ImagemLoja.java
package br.dpacanhella.miguelopolis.data.model;

import java.io.Serializable;

import lombok.Data;

/**
 * Created by infra on 05/07/17.
 */

@Data
public class ImagemLoja implements Serializable{

    private int id;
    private String descricao;
    private String imagem;
    private String image64;

}

File: iOS/miguelopolis/ImagemLoja.swift
//
//  ImagemLoja.swift
//  miguelopolis
//
//  Created by infra on 16/08/17.
//  Copyright © 2017 pacanhella. All rights reserved.
//

import Foundation
import ObjectMapper

struct ImagemLoja {
    
    var descricao: String?
    var imagem: String?
    
    
}

// MARK: - JSON Mapper
extension ImagemLoja: Mappable {
    init?(map: Map) {
        
    }
    
    mutating func mapping(map: Map) {
        descricao  <- map["descricao"]
        imagem    <- map["image64"]
        
    }
}
